offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -177,18 +170,41 @@ class OfferToPDFConverter implements OfferExporter {
         //and remove the footer on the first page
         //htmlContent.getElementById("grid-table-footer").remove()
 
-        List<ProductItem> listOverheadItems = offer.itemsWithOverhead
-        List<ProductItem> listNoOverheadItems = offer.itemsWithoutOverhead
+        List<ProductItem> productItems = offer.items
+
+        List<ProductItem> dataGenerationItems = []
+        List<ProductItem> dataAnalysisItems = []
+        //Project Management and Data Storage are grouped in the same category in the final Offer PDF
+        List<ProductItem> dataManagementItems = []
+
+        // Sort ProductItems into "DataGeneration", "Data Analysis" and "Project & Data Management"
+        productItems.each {
+            if (it.product instanceof Sequencing) {
+                dataGenerationItems.add(it)
+            }
+            if (it.product instanceof PrimaryAnalysis || it.product instanceof SecondaryAnalysis) {
+                dataAnalysisItems.add(it)
+            }
+            if (it.product instanceof DataStorage || it.product instanceof ProjectManagement){
+                dataManagementItems.add(it)
+            }
+        }
```

Comment:
I think this can be more expressive. For example call a method that expects for items and return a map with items sorted by category. 

```
Map itemsByGroup = groupItems(productItems)
```

Reply:
Good suggestion!